app/models/katello/docker_manifest.rb

```diff
@@ -5,13 +5,18 @@ class DockerManifest < Katello::Model
     has_many :docker_manifest_list_manifests, :class_name => "Katello::DockerManifestListManifest",
              :dependent => :delete_all, :inverse_of => :docker_manifest
     has_many :docker_manifest_lists, :through => :docker_manifest_list_manifests, :inverse_of => :docker_manifests
+    has_many :content_facets, :class_name => "::Katello::Host::ContentFacet", :as => :manifest_entity, :dependent => :nullify
+    has_many :hosts, :class_name => "::Host::Managed", :through => :content_facets, :inverse_of => :docker_manifest
 
     CONTENT_TYPE = "docker_manifest".freeze
 
+    scope :bootable, -> { where(:is_bootable => true) }
+
     scoped_search :relation => :docker_tags, :on => :name, :rename => :tag, :complete_value => true
     scoped_search :on => :digest, :rename => :digest, :complete_value => true, :only_explicit => true
     scoped_search :on => :schema_version, :rename => :schema_version, :complete_value => true, :only_explicit => true
     scoped_search :relation => :docker_manifest_lists, :on => :digest, :rename => :manifest_list_digest, :complete_value => true, :only_explicit => true
+    scoped_search :on => :is_bootable, :rename => :bootable, :complete_value => { true => true, false => false }, :only_explicit => true
```

Comment:
I think the complete values need to be stringified.

```
[4] pry(main)> ::Katello::DockerManifest.search_for('bootable = true')
ScopedSearch::QueryNotSupported: 'is_bootable' should be one of 'true, false', but the query was 'true'
from /home/vagrant/foreman/.vendor/ruby/3.0.0/gems/scoped_search-4.1.12/lib/scoped_search/query_builder.rb:179:in `translate_value'

```

Reply:
I tried
```rb
:complete_value => { true => "true", false => "false" }
```

but was getting the same error.
```rb
:complete_value => true
```

seems to work fine, though. Are you ok with that?

Reply:
We resolved this -- it was related to hosts without any content facet.